docs/start/building.md

```diff
@@ -50,7 +50,11 @@ mvn clean install -DskipTests
 
 This instructs [Maven](http://maven.apache.org) (`mvn`) to first remove all existing builds (`clean`) and then create a new Flink binary (`install`).
 
-To speed up the build you can skip tests, checkstyle, and JavaDocs: `mvn clean install -DskipTests -Dmaven.javadoc.skip=true -Dcheckstyle.skip=true`.
+To speed up the build you can skip tests, checkstyle, and JavaDocs:
+
+{% highlight bash %}
+mvn clean install -DskipTests -Dmaven.javadoc.skip=true -Dcheckstyle.skip=true
```

Comment:
replace `-Dmaven.javadoc.skip=true -Dcheckstyle.skip=true` with `-Dfast`

Reply:
A little question, what is the meaning of the `-Dfast` ?

Reply:
https://github.com/apache/flink/blob/56df6904688642b1c8f9a287646c163dfae7edfd/pom.xml#L639